src/swap/utils/buildSwapTransaction.ts

```diff
@@ -23,7 +23,14 @@ export async function buildSwapTransaction(
     isAmountInDecimals: false,
   };
 
-  const apiParams = getAPIParamsForToken({ ...defaultParams, ...params });
+  let apiParams = getAPIParamsForToken({ ...defaultParams, ...params });
+
+  if (!params.aggregator) {
+    apiParams = {
+      v2Enabled: true,
```

Comment:
oh I see you want to test this on his own for a bit?

Reply:
yeah, we'll do a switchover once the V2 service is more production ready